urllib3/connection.py

```diff
@@ -195,10 +195,17 @@ class VerifiedHTTPSConnection(HTTPSConnection):
     Based on httplib.HTTPSConnection but wraps the socket with
     SSL certification.
     """
-    cert_reqs = None
     ca_certs = None
     assert_fingerprint = None
 
+    @property
```

Comment:
-1 on magical properties if we can avoid them (maybe purely for backwards compat). Can we get rid of using these properties if we're going to depend entirely on the ssl_context?